practice/completer/site.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var completerPrototype = {
+  addCompletion: function(comp) {
+    this.completions.push(comp);
```

Comment:
Probably a good thing to think about, although I don't think it's totally necessary to solve the basic problem. Simple enough with ```_.includes``` that I might as well use it, thanks!